joomla/joomla-cms
# Improve new menu field

### Summary of Changes
Improve parameter loading to have real default values.

### Testing Instructions
It's a bit hard to test. You have to load the com_menus/models/fields/modal/menu.php field in your xml or took an existing one like: https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_menus/models/forms/item_alias.xml

After that you have to play with the parameters:
```
select="true"
new="true"
edit="true"
clear="true"
```

at the end all the old combinations should still work + if you remove this four parameters it should be the same as:
```
select="true"
new="false"
edit="false"
clear="true"
```
